extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -238,12 +239,24 @@ export class ConsentUI {
    * @return {!Element}
    */
   createPlaceholder_() {
-    // TODO(@zhouyx): Allow publishers to provide placeholder upon request
     const placeholder = this.parent_.ownerDocument.createElement('placeholder');
```

Comment:
I'm not sure if it is too late to change this, but you shouldn't use a tag name like `placeholder`. A browser could add one with certain semantics that might be a problem. You should only ever use valid html tag names or include a dash in the name.

Reply:
I agree we should change this.

However, @zhouyx do you see any problem in changing this? Is there more code already in place around this idea?